templates/snippets/header.html

```diff
@@ -49,25 +51,27 @@ <h2 class="my-0 mr-md-auto font-weight-normal" >
             </li> -->
             {% if user.is_authenticated %}
             <li class="nav-item">
-                <a class="btn btn-outline-success mr-sm-3 mb-2 mt-3 my-sm-0" href="{% url 'accounts:password_change' %}">Change Password</a>
+                <a class="btn btn-outline-success mr-sm-3 mb-2 mt-3 my-sm-0"
+                    href="{% url 'home:dashboard' %}">Dashboard</a>
             </li>
 
             <!-- Used in Feedbacks Page -->
             {% if logout_redirect_site %}
             <li class="nav-item">
-                <a class="btn btn-outline-danger" href="{% url 'accounts:logout' %}?next={{ logout_redirect_site }}">Sign Out</a>
+                <a class="btn btn-outline-danger"
+                    href="{% url 'accounts:logout' %}?next={{ logout_redirect_site }}">Sign Out</a>
             </li>
             {% else %}
             <li class="nav-item">
                 <a class="btn btn-outline-danger" href="{% url 'accounts:logout' %}">Sign Out</a>
             </li>
             {% endif %}
-            
+
             {% else %}
             <li class="nav-item">
                 <a class="btn btn-outline-danger" href="{% url 'accounts:login_signup' %}">Sign In</a>
             </li>
```

Comment:
Can you also add a link here to the home page, just like the dashboard link above, which will be displayed if the user is not logged in? (Dashboard link is displayed only if the user is logged in.)

Reply:
ok, I will do it